runtime/experimental/makefiles/apcu.mk

```diff
@@ -0,0 +1,23 @@
+.DEFAULT_GOAL := make_apcu
+
+url_apcu = https://github.com/krakjoe/apcu/archive/v${VERSION_APCU}.tar.gz
+upper_apcu = $(shell echo apcu| awk '{print toupper($0)}')
+build_dir_apcu = ${DEPS}/apcu
```

Comment:
Those variables used in all the `*.mk` files make it very hard to follow. Can we inline them to simplify further? At least those that are only used once.

Reply:
Let's revisit that later!